Offre.Test/User/UserLogicTests.cs

```diff
@@ -12,77 +15,194 @@ namespace Offre.Test.User
     [TestClass]
     public class UserLogicTests
     {
-        private readonly Mock<DbSet<UserModel>> _userSetMock = new Mock<DbSet<UserModel>>();
         private readonly Mock<IOffreContext> _offreContextMock = new Mock<IOffreContext>();
 
         private UserLogic GetTestSubject()
         {
             return new UserLogic(_offreContextMock.Object);
         }
 
-        //[TestMethod]
-        //public void GetAllUsers()
-        //{
-        //    var userLogic = GetTestSubject();
+        [TestMethod]
+        public void GetAllUsers_GetAllUsersFromTable_ReturnsValidUserCount()
```

Comment:
Test methods have to be async - in this way we skip x.Result method which is in most cases a bad idea and should be omitted.

Reply:
Done